package.json

```diff
@@ -46,6 +46,7 @@
         "lint:knip": "knip",
         "lint": "run-p lint:*",
         "test": "c8 mocha --reporter dot \"test/*.mjs\"",
+        "debug": "mocha --reporter dot \"test/*.mjs\"",
```

Comment:
How about `test` and `test-coverage` instead.

Reply:
Great idea! I changed the script names.